client/jetpack-cloud/sections/pricing/jpcom-masterbar/index.tsx

```diff
@@ -44,140 +54,12 @@ const JetpackComMasterbar: React.FC< Props > = ( { pathname } ) => {
-							{
-								label: translate( 'Site Search' ),
-								tagline: translate( 'Help them find what they need' ),
-								href: `${ JETPACK_COM_BASE_URL }/upgrade/search/`,
-							},
-							{
-								label: translate( 'Boost' ),
-								tagline: translate( 'Instant speed and SEO' ),
-								href: `${ JETPACK_COM_BASE_URL }/boost/`,
-							},
-							{
-								label: translate( 'VideoPress' ),
-								tagline: translate( 'High-quality, ad-free video' ),
-								href: `${ JETPACK_COM_BASE_URL }/videopress/`,
-							},
-						],
-					},
-					{
-						label: translate( 'Growth' ),
-						tagline: translate( 'Grow your audience' ),
-						href: `${ JETPACK_COM_BASE_URL }/features/growth/`,
-						items: [
-							{
-								label: translate( 'AI Assistant', { context: 'Jetpack product name' } ),
-								tagline: translate( 'Write smarter, not harder' ),
-								href: `${ JETPACK_COM_BASE_URL }/ai/`,
-							},
-							{
-								label: translate( 'Stats', { context: 'Jetpack product name' } ),
-								tagline: translate( 'Simple, yet powerful analytics' ),
-								href: `${ JETPACK_COM_BASE_URL }/stats/`,
-							},
-							{
-								label: translate( 'Social', { context: 'Jetpack product name' } ),
-								tagline: translate( 'Write once, post everywhere' ),
-								href: `${ JETPACK_COM_BASE_URL }/social/`,
-							},
-							{
-								label: translate( 'CRM' ),
-								tagline: translate( 'Connect with your people' ),
-								href: 'https://jetpackcrm.com/?utm_medium=automattic_referred&utm_source=jpcom_header',
-							},
-							{
-								label: translate( 'Blaze' ),
-								tagline: translate( 'Advertise your best content' ),
-								href: `${ JETPACK_COM_BASE_URL }/blaze/`,
-							},
-							{
-								label: translate( 'Newsletter' ),
-								tagline: translate( 'Write once, reach all' ),
-								href: `${ JETPACK_COM_BASE_URL }/newsletter/`,
-							},
-						],
-					},
-				],
-			},
-			{
-				label: translate( 'Pricing' ),
-				href: `${ JETPACK_COM_BASE_URL }/pricing/`,
-			},
-			{
-				label: translate( 'Partners' ),
-				href: `${ JETPACK_COM_BASE_URL }/jetpack-partners/`,
-			},
-			{
-				label: translate( 'Support' ),
-				href: `${ JETPACK_COM_BASE_URL }/support/`,
-			},
-			{
-				label: translate( 'Blog' ),
-				href: `${ JETPACK_COM_BASE_URL }/blog/`,
-			},
-		],
-		[ translate ]
-	);
-	const bundles = useMemo(
-		() => ( {
-			label: translate( 'Bundles' ),
-			items: [
-				{
-					label: translate( 'Complete', { context: 'Jetpack plan name' } ),
-					tagline: translate( 'The ultimate toolkit' ),
-					href: `${ JETPACK_COM_BASE_URL }/complete/`,
-				},
-				{
-					label: translate( 'Security', { context: 'Jetpack product name' } ),
-					tagline: translate( 'Comprehensive site security' ),
-					href: `${ JETPACK_COM_BASE_URL }/features/security/`,
-				},
-			],
-		} ),
-		[ translate ]
-	);
+	const sortByMenuOrder = ( a: MenuItem, b: MenuItem ) => a.menu_order - b.menu_order;
```

Comment:
I think something is still wrong with order - I suppose "Products" should be first in the menu, analogically to jetpack.com menu:
<img width="1285" alt="CleanShot 2023-08-14 at 16 53 29@2x" src="https://github.com/Automattic/wp-calypso/assets/8419292/211108e3-7985-40f5-82ee-fbc6357b6d2e">

Reply:
That is strange, the menu_order of Products is correct, and I have only ever seen it be the first item in the menu since I've sorted it that way. I cannot reproduce this but it is concerning that you see it that way 🤔

Reply:
I also used the `/fr/` localizer to get the GDPR link to show up, but it is still in the correct order for me 🤔